tchannel-core/src/main/java/com/uber/tchannel/api/Request.java

```diff
@@ -76,18 +82,23 @@ public String toString() {
 
     public static class Builder<U> {
 
+        private static final long DEFAULT_TTL = 100;
```

Comment:
nitpick: this contant makes little sense, you can just have the default value of var ttl itself as 100.